dynamic_programming/longest_common_subsequence/longest_common_subsequence_test.go

```diff
@@ -0,0 +1,35 @@
+package main
+
```

Comment:
please eliminate the blank lines

Reply:
There should be blank lines between package name and import. I can't find official documentation for now but I am attaching the digitalOcean tutorials. You can check the format of Go code.
https://www.digitalocean.com/community/tutorials/importing-packages-in-go